contrib/epee/include/net/levin_protocol_handler.h

```diff
@@ -1,193 +0,0 @@
-// Copyright (c) 2006-2013, Andrey N. Sabelnikov, www.sabelnikov.net
```

Comment:
Yeah I can remove these changes to this specific file, I just added the commit from that PR for removing the `core_proxy` tests since I didn't want reviewers to have to review the downstream changes to the `core_proxy` tests which are going to be removed anyways.